web/packages/teleport/src/UnifiedResources/ResourceCard.tsx

```diff
@@ -261,9 +261,34 @@ export function ResourceCard({ resource, onLabelClick }: Props) {
   );
 }
 
-export function LoadingCard() {
-  function randomNum(min: number, max: number) {
-    return Math.floor(Math.random() * (max - min + 1)) + min;
+type LoadingCardProps = {
+  delay?: 'none' | 'short' | 'long';
+};
+
+const DelayValueMap = {
+  none: 0,
+  short: 400, // 0.4s;
+  long: 600, // 0.6s;
+};
+
+function randomNum(min: number, max: number) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
```

Comment:
This isn't new, i just extracted it from the component since it didn't need to be in there.